orchestra/bots/tests/test_views.py

```diff
@@ -37,24 +51,167 @@ def test_post_invalid_data(self):
         response = self.request_client.post(self.url, data)
         self.assert_response(response, error=True)
 
-    def test_staff_command(self):
+    @patch('orchestra.bots.staffbot.StaffBot.send_staffing_request_by_mail')
+    @patch('orchestra.bots.staffbot.StaffBot.send_staffing_request_by_slack')
+    def test_staff_command(self, mock_slack, mock_mail):
```

Comment:
can you break up the difference cases in to separate unit tests? it's easier to read/know what is being tested